File: BEE 1113.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        while (true) {
            int x = input.nextInt();
            int y = input.nextInt();
            if (x == y){
                break;
            }
    
            if (x > y){
                System.out.println("Decrescente");
            }
            else{
                System.out.println("Crescente");
            }
        }
    }
}

File: BEE 1113.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    while(1){
        int x, y;
        cin >> x >> y;
        if (x == y){
            break;
        }

        if (x > y){
            cout << "Decrescente" << endl;
        }
        else{
            cout << "Crescente" << endl;
        }
    }
    return 0;
}